packages/plugin/src/Fields/NumberField.php

```diff
@@ -74,24 +64,33 @@ public function getValue(): mixed
         return $value;
     }
 
+    public function getMinMaxValues(): array
+    {
+        if (!\is_array($this->minMaxValues)) {
+            return [];
+        }
+
+        return $this->minMaxValues;
+    }
+
     public function getType(): string
     {
         return self::TYPE_NUMBER;
     }
 
-    public function getMinLength(): int
+    public function getMinLength(): ?int
     {
         return $this->minLength;
     }
 
     public function getMinValue(): int
     {
-        return $this->minValue;
+        return $this->minMaxValues[0];
```

Comment:
This would die currently without having the values specified, since the default return value is `[]` instead of `[null, null]`

And maybe it would be prettier to do something like this:

```suggestion
        [$min] = $this->getMinMaxValues();

        return $min;
```